src/arduino.cc/builder/i18n/errors.go

```diff
@@ -0,0 +1,20 @@
+package i18n
+
+import "arduino.cc/builder/constants"
+import "github.com/go-errors/errors"
+import "os"
+
+func ErrorfWithLogger(logger Logger, format string, a ...interface{}) *errors.Error {
+	if logger.Name() == "machine" {
+		logger.Fprintln(os.Stderr, constants.LOG_LEVEL_ERROR, format, a...)
+		return errors.Errorf("")
+	}
+	return errors.Errorf(Format(format, a...))
+}
+
+func WrapError(err error) error {
+	if err == nil {
+		return nil
+	}
+	return errors.Wrap(err, 0)
+}
```

Comment:
Not really related, but I've been forced to do this otherwise there is a cyclic import. The problem is in the `ErrorfWithLogger` method that wants a `Logger`, but the `Logger` interface is defined in `i18n` package, so the cyclical dependency.

For the moment I placed everything into the `i18n` package that is the solution with the minimum amount of changes. We can find a better way after merging the PR.